feat(web-core): new layout system

ByScripts opened the pull request:
### Description

New layout system with a default `CoreLayout` component which handle header + resizable sidebar + content header + content + right panel header/content.

![layout](https://github.com/vatesfr/xen-orchestra/assets/19408/80291109-1c0d-4091-b156-576f7030cbae)


### Checklist

- Commit
  - Title follows [commit conventions](https://bit.ly/commit-conventions)
  - Reference the relevant issue (`Fixes #007`, `See xoa-support#42`, `See https://...`)
  - If bug fix, add `Introduced by`
- Changelog
  - If visible by XOA users, add changelog entry
  - Update "Packages to release" in `CHANGELOG.unreleased.md`
- PR
  - If UI changes, add screenshots
  - If not finished or not tested, open as _Draft_

ByScripts committed:
feat(web-core): new layout components

OlivierFL commented on @xen-orchestra/web-core/lib/components/layout/LayoutSidebar.vue:
```diff
@@ -0,0 +1,99 @@
+}
+
+.resize-handle {
+  content: '';
```
Not needed.

OlivierFL commented on @xen-orchestra/web-core/lib/layouts/CoreLayout.vue:
```diff
@@ -0,0 +1,184 @@
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
```
What do you think of replacing these lines with `inset`?
```suggestion
  inset: 0;
```

ByScripts committed:
update layout

Rename "pin" to "lock"
Add i18n
Replace top/right/bottom/left with inset
Add TooltipList to App.vue

ByScripts commented:
@pdonias @MathieuRA Can you merge? Thanks.
